db/seeds.rb

```diff
@@ -1,8 +1,11 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-#
+AdminUser.create(email: 'admin@example.com', password: 'password')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.